transport/src/main/java/io/netty/channel/nio/AbstractNioByteChannel.java

```diff
@@ -219,13 +219,11 @@ protected void doWrite(ChannelOutboundBuffer in) throws Exception {
                 if (done) {
                     in.remove();
                 } else {
-                    incompleteWrite(setOpWrite);
                     break;
```

Comment:
@normanmaurer - Sure that is fine...more of just me rushing this time around. I got it the first time :)